.circleci/config.yml

```diff
@@ -116,6 +124,10 @@ jobs:
                 submit:true \
                 api_token:$HOCKEY_APP_TOKEN \
                 app_id:$HOCKEY_APP_IOS_ALPHA_APP_ID \
+                fabric_api_token:$FABRIC_API_KEY \
+                fabric_build_secret:$FABRIC_BUILD_SECRET \
+                fabric_groups:$FABRIC_GROUPS \
```

Comment:
Groups are a list of Fabric created groups of testers allowed to view the build in Fabric